docs/README.md

```diff
@@ -88,6 +91,29 @@ CAROOT=$(pwd)/.certs mkcert -install
 
 For windows, we recommend installing the root certificate via the [MMC snap-in](https://learn.microsoft.com/en-us/troubleshoot/windows-server/windows-security/install-imported-certificates#import-the-certificate-into-the-local-computer-store).
 
+## Deploying the Apexctl Utility
```

Comment:
```suggestion
## Using the Apexctl Utility
```

How about "Using" instead of "Deploying."  I suggest that because I don't think of CLI utilities as something deployed.